qt/aqt/import_export/exporting.py

```diff
@@ -86,7 +86,7 @@ def setup(self, did: DeckId | None) -> None:
             self.frm.includeSched.setChecked(False)
 
     def exporter_changed(self, idx: int) -> None:
-        self.exporter = self.exporters[idx]
+        self.exporter = self.exporter_classes[idx]()
```

Comment:
There are a couple of add-ons that reference exporters - is it worth breaking them to rename this?

Reply:
(otherwise looks good)

Reply:
No, no need to cause any extra breakages I'd say. Pushed a change to revert the naming.

However, out of curiosity: Were the add-ons you found add-ons that already modify the new `ExportDialog` implementation? I did try grepping for potentially affected add-ons before the PR, but the ones I found were targeting the old dialog, and correct me if I'm wrong, but they would have to be updated anyway to support the new code, right? (i.e. there's no reference that Anki used to keep to the old `ExportDialog` (e.g. via the dialog manager) that's now switched out to the new one and could trip up add-ons if they access `.exporters`?)

If we aren't breaking compatibility of existing add-ons, but just introducing an extra step on top of other necessary adjustments, then I would slightly favor going with `exporter_classes` as it highlights the difference to `.exporter` better. But this is super minor, and at the end of the day both would be absolutely fine by me.

Reply:
You are right - I just did a quick grep without really thinking about it, but this is new code that's unlikely to have been used yet. Sorry for the wasted effort; feel free to revert the revert.

Reply:
No worries, thanks for checking! ok, done